flow/table.go

```diff
@@ -582,14 +584,18 @@ func (ft *Table) Run() {
 			if ft.ipDefragger != nil {
 				ft.ipDefragger.FlushOlderThan(t)
 			}
-		case <-statusTicker.C:
-			ft.status.CaptureID = ft.uuids.CaptureID
-			ft.status.FlowCount = int64(ft.table.Len())
-
-			ft.sender.SendStatus(ft.status)
-
-			logging.GetLogger().Warningf("Flow table status: %+v", ft.status)
-			ft.status = Status{}
+		case s := <-ft.statsChan:
+			ft.stats.KernelFlowDropped += s.KernelFlowDropped
+			ft.stats.PacketsReceived += s.PacketsReceived
+			ft.stats.PacketsDropped += s.PacketsDropped
+		case <-statsTicker.C:
+			ft.stats.CaptureID = ft.uuids.CaptureID
+			ft.stats.FlowCount = int64(ft.table.Len())
+
+			ft.sender.SendStats(ft.stats)
+
+			logging.GetLogger().Warningf("Flow table stats: %+v", ft.stats)
```

Comment:
Why is this a warning? Should be info.

Reply:
u probably want to delete this log message entirly

Reply:
just to keep to previous behavior, I'll add it as debug